core/src/main/java/org/infinispan/persistence/sifs/NonBlockingSoftIndexFileStore.java

```diff
@@ -156,6 +157,7 @@
 
    @Override
    public Set<Characteristic> characteristics() {
+      // Does not support segmented or expiration yet
```

Comment:
Hey, I thought you were improving the already existing segmentation support here :)

Reply:
Yeah, that snuck in somehow, haha.